### DISCUSSION TITLE
Is there a way to extract styles from Tailwind classes for snapshot testing?

### DISCUSSION THREAD

**Discussion by freddydumont** [tailwindlabs/tailwindcss#4761] [Help]
In the CSS-in-JS world, we can [serialize the styles](https://emotion.sh/docs/testing) for snapshot testing. Is there a way to achieve the same result with Tailwind? 

For example, rather than snapshotting this:

```jsx
<code class="text-sm font-bold text-gray-900">tailwind.config.js</code>
```

I'd have something like this:

```jsx
.tailwind-1 {
  font-size: 0.875rem/* 14px */;
  line-height: 1.25rem/* 20px */;
  font-weight: 700;
  --tw-text-opacity: 1;
  color: rgba(17, 24, 39, var(--tw-text-opacity));
}

<code class="tailwind-1">tailwind.config.js</code>
```

tldr; is there a jest snapshot serializer for tailwind?

**Comment by thiagonzalez** [comment]
I have the same question, @freddydumont.
